.travis.yml

```diff
@@ -1,13 +1,17 @@
-language: julia
-os:
-    - linux
-    - osx
-julia:
-    - 0.6
-    - nightly
-notifications:
-    email: false
-sudo: false
+ language: julia
+
+ os:
+     - linux
+     - osx
+
+ julia:
+    - 0.7
+    - 1.0
+
+ notifications:
+     email: false
+
+ sudo: false
 script:
     - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
     - julia -e 'Pkg.clone(pwd())'
```

Comment:
Remember to remove this block,
There are a default script from the julia Travis integration in 0.7/1.0.

Reply:
Done